File: Arrays/Python/Alternative_Sorting.py
class Solution:
    def alternateSort(self,arr, N):
        # Your code goes here
        arr.sort()

        l=[]

        i=0

        j=N-1

        while i<j:

            l.append(arr[j])

            j-=1

            l.append(arr[i])

            i+=1

        if N%2!=0:

            l.append(arr[i])
        return l

File: Solutions/Java/Alternative_Sorting.java
class Solution
{
    ArrayList<Long> alternateSort(long arr[] ,int N)
    {
        
        // Your code goes here
        ArrayList<Long> list=new ArrayList<>(); 
        Arrays.sort(arr);
        int start=0;
        int end=N-1;

        while (start<=end) {
            list.add(arr[end]);
            list.add(arr[start]);
            start++;
            end--;
        }
        return list;
    }
}
